app/templates/index.hbs

```diff
@@ -6,6 +6,20 @@
   code firing such deprecations is still supported by the Ember community. After
   the next major revision lands, the supporting code may be removed. This style
   of change management is commonly referred to as <a href="http://semver.org/">Semantic Versioning</a>.</p>
+  
+  <p>
+    <a href="https://github.com/ember-cli/ember-cli/releases">Upgrade your ember-cli version</a> 
```

Comment:
This isn't correct, what contains the deprecations is the Ember source code, not Ember CLI.